library/Zend/Code/Generator/ValueGenerator.php

```diff
@@ -239,7 +318,7 @@ public function generate()
                 );
                 foreach ($rii as $curKey => $curValue) {
                     if (!$curValue instanceof ValueGenerator) {
-                        $curValue = new self($curValue);
+                        $curValue = new self($curValue, self::TYPE_AUTO, self::OUTPUT_MULTIPLE_LINE, $this->getConstants());
```

Comment:
Constants is a very big array, containing about 1k string values. Let's length of every string length is 10, therefore total  size of array in memory would be greater then 10KB for one instance of ValueGenerator, that means for 1 property of generated class. So for 100 classes with 100 properties total size of constant arrays would be 10MB instead of 10KB for reference value